transformer_engine/pytorch/module/layernorm_mlp.py

```diff
@@ -344,7 +344,7 @@ def forward(
                 dim_size = list(gelu_out.size())
                 dim_size[1] = fc2_weight.size(0)
                 fc2_out = torch.empty(dim_size, dtype=activation_dtype, device=gelu_out.device)
-            _, _, _ = tex.gemm(
+            _ = tex.gemm(
```

Comment:
Nitpick:
```suggestion
            tex.gemm(
```
I see this pattern in a few other places.

Reply:
Right, the linter complaints about return value not received for that case